db/migrate/20190617100508_add_limit_exceeded_error_to_proxies.rb

```diff
@@ -0,0 +1,7 @@
+class AddLimitExceededErrorToProxies < ActiveRecord::Migration
+  def change
+    add_column :proxies, :error_headers_limits_exceeded, :string, default: 'text/plain; charset=us-ascii'
```

Comment:
This is still blocking 😞 
![image](https://user-images.githubusercontent.com/11318903/59836038-255a0e80-934b-11e9-96c7-060e015fd9ce.png)

I propose to do as it says there but the 2nd migration to be:
```ruby
class BackfillAddLimitExceededErrorToProxies < ActiveRecord::Migration
  def up
    Proxy.find_in_batches(batch_size: 100) do |batch|
      batch.each { |proxy| proxy.update_column(error_headers_limits_exceeded: 'text/plain; charset=us-ascii') }
      sleep(1) # So we let customers do requests to the DB instead of blocking it for long. This is because it needs to work for all DBs
    end
  end

  def down
    Proxy.find_in_batches(batch_size: 100) do |batch|
      batch.each { |proxy| proxy.update_column(error_headers_limits_exceeded: nil) }
      sleep(1) # So we let customers do requests to the DB instead of blocking it for long. This is because it needs to work for all DBs
    end
  end
end
```

Reply:
why find in batches and not `Proxy.update_all` ? without loading everything..

Reply:
Because when I did [the PR of the rake tasks to fix empty or corrupted tenant_id in the DB](https://github.com/3scale/porta/pull/813), I made some tests with operations in preview, and doing `update_all` for literally all the records of a table means it takes very long and in that time the requests of customers are waiting because these are locked... 😞 so we changes it to be like this (updating one by one so customer requests can take a spot in between those) and with a waiting time of 1 second without doing anything so that time is only for customers.
But you should probably speak with operations to see if that is ideal for this case too or if they have a better idea in mind 😄